app/policies/subject_policy.rb

```diff
@@ -9,9 +9,7 @@ def manage_assessments?
 
   class Scope < Scope
     def resolve
-      Subject.
-        with_direct_assessments.
-        where(department_number: user.departments.pluck(:number))
+      Subject.where(department_number: user.departments.pluck(:number))
```

Comment:
Will opening this scope up mean that when you go to add results, all subjects will display, not just those with direct assessments?